pkg/apis/aws/v1alpha1/account_types.go

```diff
@@ -50,8 +50,10 @@ type AccountStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
-	Claimed                  bool               `json:"claimed,omitempty"`
-	SupportCaseID            string             `json:"supportCaseID,omitempty"`
+	Claimed       bool   `json:"claimed,omitempty"`
+	SupportCaseID string `json:"supportCaseID,omitempty"`
+	// +listType=map
+	// +listMapKey=type`
```

Comment:
FYI this nonsense will go away in a subsequent upgrade step

Reply:
That was frustrating to track down, too.  :+1:

Reply:
Yeah, took me a good bit of time when I was doing gcp-project-operator. They could have done a better job documenting it.